v21.2/cost-based-optimizer.md

```diff
@@ -102,6 +102,16 @@ SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
 
 When `sql.stats.histogram_collection.enabled` is set to `false`, histograms are never collected, either as part of automatic statistics collection or by manually invoking [`CREATE STATISTICS`](create-statistics.html).
 
+## Control whether the optimizer creates a plan with a full scan
+
+Even if you have [secondary indexes](schema-design-indexes.html), the optimizer may decide that a full table scan will be faster. For example, if you add a secondary index to a table with a large number of rows and determine that a statement plan isn't using the secondary index, it is likely that performing a full table scan using the primary key is faster than doing a secondary index scan plus an [index join](indexes.html#example).
```

Comment:
```suggestion
Even if you have [secondary indexes](schema-design-indexes.html), the optimizer may determine that a full table scan will be faster. For example, if you add a secondary index to a table with a large number of rows and find that a statement plan is not using the secondary index, then it is likely that performing a full table scan using the primary key is faster than doing a secondary index scan plus an [index join](indexes.html#example).
```

Reply:
I was slightly slow to parse the second sentence, so am making these suggestions for readability. Feel free to disregard. However, I feel that "determine" is better than "decide" because the latter can imply (at least to me) that the optimizer's choice might be arbitrary and/or incorrect.